src/Ombi.Core/Engine/Demo/DemoMovieSearchEngine.cs

```diff
@@ -50,7 +50,7 @@ public async Task<IEnumerable<SearchMovieViewModel>> Search(string search)
             return null;
         }
 
-        new public async Task<IEnumerable<SearchMovieViewModel>> NowPlayingMovies()
+        public async Task<IEnumerable<SearchMovieViewModel>> NowPlayingMovies()
```

Comment:
these changes seem unrelated?

Reply:
Indeed, bad rebase, not sure how I ended up modifying that file...